README.rst

```diff
@@ -503,8 +503,83 @@ Models
 ======
 
 Models are statistical analyses that are carried out on the features.
+Often these are known as second-level or group-level analyses.
+There are currently two ways in which to run these analyses, either through
+the user interface, or using a ‘group-level’ command directly in the terminal.
+Shown first is the user interface method.
+
+#. ``Specify group-level model?``
+
+   -  ``Yes``
+
+      #. ``Specify model type``
+
+         -  ``Intercept-only`` This will apply an intercept-only model that captures a main effect of one or more of the first-level features.
+
+            #. ``Specify model name``
+            #. ``Aggregate scan-level statistics before analysis?`` This option may appear if there are multiple subject-level statistical images created, such as for multiple runs or sessions. If this option is selected (with an asterisk) then these features will be aggregated at the subject-level prior to group-level analysis. If the option is not selected, then all of the statistical images of each subject’s first-level features will be entered into the group-level analysis, treating them as if they were independent observations from different subjects, therefore this option is generally not preferred.
+            #. ``Exclude subjects based on movement?``
+
+               -  ``Yes``
+
+                  #. ``Specify the maximum allowed mean framewise displacement in mm`` By selecting 0.5, this will exclude all subjects (or runs/sessions for a subject if ‘aggregation of scan-level statistics before analysis’ is not selected above) with a mean framewise displacement (root-mean-square) >0.5 mm. If aggregation of scan-level statistics before analysis is selected, the aggregate image will be used in the group-level analyses, therefore the average framewise displacement of all images used for the aggregate image will be compared to the value set here – if it is exceeded, the subject will be excluded from analysis.
```

Comment:
The threshold will always be applied to individual runs/sessions, so that if a subject has one bad run, the other can still enter the group analysis

Reply:
Resolved